guide/05-working-with-the-spatially-enabled-dataframe/part2_data_io_reading_data.ipynb

```diff
@@ -0,0 +1,5229 @@
+{
```

Comment:
<p>The <code><strong>from_featureclass()</strong></code> method allows users to specify optional parameters when the <code>ArcPy</code> library is available in the current environment. These parameters are: </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>